lib/webpacker/file_loader.rb

```diff
@@ -7,9 +7,17 @@ class FileLoaderError < StandardError; end
   attr_accessor :data
 
   class << self
+    def exist?
+      instance.data.present? rescue false
+    end
+
     def load(path = file_path)
       self.instance = new(path)
     end
+
+    def reloadable?
+      Rails.configuration.x.webpacker[:reload]
```

Comment:
Instead of coupling webpacker with the Rails, we should be pushing the configuration to `Webpacker::FileLoader` from inside the Railtie.

```ruby
initializer :webpacker do |app|
  Webpacker::FileLoade.reloadable = app.config.webpacker.reload
```